DrivewayDentDeletion/Operators/apic-resources/cicd-apply-test-apic-pipeline.sh

```diff
@@ -0,0 +1,238 @@
+#!/bin/bash
+#******************************************************************************
+# Licensed Materials - Property of IBM
+# (c) Copyright IBM Corporation 2019. All Rights Reserved.
+#
+# Note to U.S. Government Users Restricted Rights:
+# Use, duplication or disclosure restricted by GSA ADP Schedule
+# Contract with IBM Corp.
+#****
+
+# PREREQUISITES:
+#   - Logged into cluster on the OC CLI (https://docs.openshift.com/container-platform/4.4/cli_reference/openshift_cli/getting-started-cli.html)
+#
+# PARAMETERS:
+#   -n : <namespace> (string), Defaults to 'cp4i'
+#   -r : <repo> (string), Defaults to 'https://github.com/IBM/cp4i-deployment-samples.git'
+#   -b : <branch> (string), Defaults to 'master'
```

Comment:
```suggestion
#   -b : <branch> (string), Defaults to 'main'
```

Reply:
oops that script shouldn't be there, removed